packages/design-tokens/src/tokens/functional/colors/global.json

```diff
@@ -103,7 +103,7 @@
       },
       "border": {
         "default": {
-          "value": "var(--base-color-scale-gray-2)",
+          "value": "var(--base-color-scale-gray-3)",
```

Comment:
This is set as `3` in Figma, so aligning it here.. but maybe need to update some borders to use `muted` elsewhere?